lib/explorer/data_frame.ex

```diff
@@ -233,24 +233,21 @@ defmodule Explorer.DataFrame do
 
   ## Options
 
-    * `columns` - List with name of columns to be selected. Defaults to all columns.
-    * `projection` - List with the index of columns to be selected. Defaults to all columns.
+    * `with_columns` - List with the name or index of columns to be selected. Defaults to all columns.
   """
   @doc type: :io
   @spec from_ipc(filename :: String.t()) :: {:ok, DataFrame.t()} | {:error, term()}
   def from_ipc(filename, opts \\ []) do
     opts =
       Keyword.validate!(opts,
-        columns: nil,
-        projection: nil
+        with_columns: nil
       )
 
     backend = backend_from_options!(opts)
 
     backend.from_ipc(
       filename,
-      opts[:columns],
-      opts[:projection]
+      opts[:with_columns]
```

Comment:
Let’s rename the option to `columns` too!

Reply:
Ok!